files/image_config/kubernetes/scripts/kube_join.sh

```diff
@@ -0,0 +1,43 @@
+if [ ! -f /etc/sonic/kubeadm_join.sh ]
+then
+    echo "Run 'kubeadm token create --print-join-command' in Master."
+    echo "please get join command in /etc/sonic/kubeadm_join.sh"
+    exit -1
+fi
+
+if [ ! -f /etc/sonic/kube_admin.conf ]
+then
+    echo "please get kube context in /etc/sonic/kube_admin.conf"
+    echo "It can be found @ master: /etc/kubernetes/admin.conf"
+    exit -1
+fi
+
+# Reset
+modprobe br_netfilter
```

Comment:
Could we build the module into SONiC by default, even without INSTALL_KUBERNETES?

Reply:
We can do it in 2020 build. The build 20191130 is locked for new features.

Reply:
Change my ask: Could we build the module into SONiC by default, when INSTALL_KUBERNETES?

---
In reply to: [404435969](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r404435969) [](ancestors = 404435969)

Reply:
I prefer take it outside this PR, as it is not a straightforward one (unless I get some tips).

Reply:
Not a requirement for now. When required, will make it part of image.

Reply:
This whole file is removed